LVT/Classes/Program.cs

```diff
@@ -1,19 +1,34 @@
 ﻿using LVT.LVT.Services;
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 
 namespace LVT
 {
     public class Program
     {
-
         static void Main(string[] args)
         {
             //this takes an absolute path and needs to be adapted for each computer depending on where the file lives that you want to open - but this allows you to pass different template trees.
             //see Tests for an alternative way that uses project resources instead - so tests can be run no matter on whose environment
-            StreamReader file = File.OpenText(@"C:\Users\beckerfs\Documents\Projects\LVT\Lean-Value-Tree-Visualisation\LVT\SingleBranchLVT.json");
+            //will no longer be an issue when we start the program from the command line with the file location as argument
+            //StreamReader file = File.OpenText(@"C:\Users\beckerfs\Documents\Projects\LVT\Lean-Value-Tree-Visualisation\LVT\SingleBranchLVT.json");
+            StreamReader file = File.OpenText(@"C:\Users\beckerfs\Documents\Projects\LVT\Lean-Value-Tree-Visualisation\LVT\TwoGoalsTwoBetsLVT.json");
             JsonParser Parser = new JsonParser();
             LeanValueTree newTree = Parser.ParseJsonLVTFromStream(file);
+            VisionPresenter VP = new VisionPresenter();
+            string OrgaChart = VP.VisualizeToString(newTree.Vision);
+            
+            //Console.WriteLine(OrgaChart);
+
+            // this needs to be moved to separate function
+
+            Directory.CreateDirectory("C:\\temp");
+            string content = Properties.Resources.TemplatePart1 + OrgaChart + Properties.Resources.TemplatePart2;
+            File.WriteAllText(@"C:\temp\LVT.html", content);
```

Comment:
Should this be moved from the temp file to a dedicated project folder?

Reply:
We can create this where we want it.  The rationale for storing it here was that most computers will have a c drive, and many of them will already have a 'temp' folder (in which case a new one doesn't need to be created).  The location of where this is saved to can be printed on the screen for the user.